Minesweeper/Minesweeper/Entities/MineField.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  MineField.swift
+//  Minesweeper
+//
+//  Created by Dorian Grolaux on 19/02/2018.
+//
+
+import Foundation
+
+typealias Coordinates = (x: Int, y: Int)
+
+class MineField {
+    private let cells: [[Cell]]!
```

Comment:
Why make this an implicitly unwrapped optional? Non-optional seems like the reasonable choice.